moto/dynamodb2/models/__init__.py

```diff
@@ -1690,6 +1720,17 @@ def restore_table_from_backup(self, target_table_name, backup_arn):
         self.tables[target_table_name] = new_table
         return new_table
 
+    def restore_table_to_point_in_time(self, target_table_name, source_table_name):
+        source = self.get_table(source_table_name)
+        if source is None:
+            raise KeyError()
+        existing_table = self.get_table(target_table_name)
+        if existing_table is not None:
+            raise ValueError()
```

Comment:
Same here

Reply:
Done, let me know if the test looks adequate.